Sources/CombineDataSources/CollectionView/CollectionViewItemsController.swift

```diff
@@ -31,6 +32,8 @@ public class CollectionViewItemsController<CollectionType>: NSObject, UICollecti
   
   /// A fallback data source to implement custom logic like indexes, dragging, etc.
   public var dataSource: UICollectionViewDataSource?
+
+  open var configureSupplementaryView: SupplementaryViewFactory?
```

Comment:
is there a specific reason this property needs to be `open`?

Reply:
Nope, will change this to public